content/en/news/releases/1.10.x/announcing-1.10/change-notes/index.md

```diff
@@ -0,0 +1,178 @@
+release: 1.10
+weight: 10
+aliases:
+    - /news/announcing-1.10
+---
+
+## Deprecation Notices
+
+These notices describe functionality that will be removed in a future release according to [Istio's deprecation policy](/docs/releases/feature-stages/#feature-phase-definitions). Please consider upgrading your environment to remove the deprecated functionality.
+
+- **Deprecated** the `values.global.jwtPolicy=first-party-jwt` option. This option is less secure and intended for backwards compatibility
+with older Kubernetes clusters without support for more secure token authentication, but is now enabled by default in new Kubernetes versions.
+The option will be removed in future releases. See [this documentation](/docs/ops/best-practices/security/#configure-third-party-service-account-tokens) for more information.
+
+- **Deprecated** the `values.global.arch` option in favor of the affinity Kubernetes settings.
+  ([Issue #30027](https://github.com/istio/istio/issues/30027))
+
+## Traffic Management
+
+- **Added** `meshConfig.discoverySelectors` to dynamically restrict the set of namespaces for `Services`, `Pods`, and `Endpoints` that istiod processes when pushing xDS updates to improve performance on the data plane.
+  ([Blog](/blog/2021/discovery-selectors/), [Issue #26679](https://github.com/istio/istio/issues/26679))
+
+- **Added** the `ISTIO_GATEWAY_STRIP_HOST_PORT` environment variable to control whether gateways strip host port before any processing of request by HTTP filters or routing. This option is disabled by default.
+  ([Issue #25350](https://github.com/istio/istio/issues/25350))
+
+- **Fixed** configuration of TLS parameters (TLS version, TLS cipher suites, curves, etc.) with `EnvoyFilter`.
+  ([Issue #28996](https://github.com/istio/istio/issues/28996))
+
+- **Fixed** an issue where filter chain name was ignored when processing `EnvoyFilter` match.
+  ([Issue #31166](https://github.com/istio/istio/issues/31166))
+
+- **Improved** the full push scoping by adding `Sidecar` config to `sidecarScopeKnownConfigTypes`.
+
+- **Improved** virtual machine integration to clean up `iptables` rules when the service is stopped.
+  ([Issue #29556](https://github.com/istio/istio/issues/29556))
+
+- **Updated** istio-proxy drain notification strategy to immediate from gradual.
+  ([Issue #31403](https://github.com/istio/istio/issues/31403))
+
+- **Added** CNI metrics counting repair operations.
+  ([Issue #19300](https://github.com/istio/istio/issues/19300))
+
+- **Added** `/debug/connections` istiod debug interface to list the current connected clients.
+  ([Issue #31075](https://github.com/istio/istio/issues/31075))
+
+- **Added** SDS secrets fetch failure metric `pilot_sds_certificate_errors_total`.
+  ([Issue #31779](https://github.com/istio/istio/issues/31779))
+
+- **Added** metrics for istiod informer errors.
+
+- **Added** 'istioctl validate' and the validating webhook now report duplicate or unreachable virtual service matches.
```

Comment:
should this be moved to below `istioctl` section?

Reply:
Yep thanks for catching, moved it over